OpenRA.Mods.Common/Traits/BotModules/Squads/States/GroundStates.cs

```diff
@@ -57,6 +57,9 @@ public void Tick(Squad owner)
 					owner.Bot.QueueOrder(new Order("AttackMove", u, Target.FromCell(owner.World, owner.TargetActor.Location), false));
 
 				// We have gathered sufficient units. Attack the nearest enemy unit.
+				// Inform human allies about AI's rush attack.
+				owner.Bot.QueueOrder(new Order("PlaceBeacon", owner.SquadManager.Player.PlayerActor, Target.FromCell(owner.World, owner.TargetActor.Location), false)
+				{ SuppressVisualFeedback = true });
```

Comment:
Style nit: This needs one level of indentation.

Reply:
> Do we want to have the bot send a message alongside? Just a beacon might be confusing at first.

I want to but I can't find the code on how to chat in team

Reply:
ok,  a table  should enough?